controller/array_action/array_mediator_svc.py

```diff
@@ -1087,3 +1102,10 @@ def demote_replication_volume(self, replication_name):
         rcrelationship = self._get_rcrelationship_by_name(replication_name)
         endpoint_type_to_promote = self._get_replication_other_endpoint_type(rcrelationship)
         self._ensure_endpoint_is_primary(rcrelationship, endpoint_type_to_promote)
+
+    def get_host_name_if_equal(self, iscsi_host, nvme_host, fc_host):
+        s = {iscsi_host, nvme_host, fc_host}
+        s.discard(None)
+        if len(s) > 1:
+            return None
+        return s.pop()
```

Comment:
in this implementation, `get_host_name_if_equal` doesn't support all-None parameters (even though it won't happen with current `get_host_by_host_identifiers` implementation).
```suggestion
        if len(s) == 1:
            return s.pop()
        return None
```